src/theme/components/banner.ts

```diff
@@ -0,0 +1,164 @@
+import { createMultiStyleConfigHelpers } from "@chakra-ui/styled-system";
+
+import {
+  neutral as baseNeutral,
+  informative as baseInformative,
+  positive as basePositive,
+  negative as baseNegative,
+  warning as baseWarning,
+  recommendation as baseRecommendation,
+} from "./statusBadge";
+
+const { defineMultiStyleConfig, definePartsStyle } =
+  createMultiStyleConfigHelpers(["base", "dismissibleButton"]);
+
+// For now, this semantic mapping is specific to the Banner component.
+const neutral = definePartsStyle({
+  base: {
+    ...baseNeutral,
+    bgColor: "ui.bg.default",
+    svg: {
+      fill: "ui.border.hover",
+      _dark: {
+        fill: "dark.ui.border.hover",
+      },
+    },
+    _dark: {
+      bgColor: "dark.ui.bg.default",
+    },
+  },
+});
+const informative = definePartsStyle({
+  base: {
+    ...baseInformative,
+    svg: {
+      fill: "ui.link.primary",
+      _dark: {
+        fill: "dark.ui.link.primary",
+      },
+    },
+  },
+});
+const positive = definePartsStyle({
+  base: {
+    ...basePositive,
+    svg: {
+      fill: "ui.success.primary",
+      _dark: {
+        fill: "dark.ui.success.primary",
+      },
+    },
+  },
+});
+const negative = definePartsStyle({
+  base: {
+    ...baseNegative,
+    color: "ui.error.primary",
+    a: {
+      color: "ui.error.primary",
+    },
+    svg: {
+      fill: "ui.error.primary",
+      _dark: {
+        fill: "dark.ui.error.primary",
+      },
+    },
+    _dark: {
+      a: {
+        color: "dark.ui.error.primary",
+      },
+      color: "dark.ui.error.primary",
+    },
+  },
+});
+const warning = definePartsStyle({
+  base: {
+    ...baseWarning,
+    svg: {
+      fill: "ui.warning.primary",
+      _dark: {
+        fill: "dark.ui.warning.primary",
+      },
+    },
+  },
+});
+const recommendation = definePartsStyle({
+  base: {
+    ...baseRecommendation,
+    svg: {
+      fill: "ui.link.tertiary",
+      _dark: {
+        fill: "dark.ui.link.tertiary",
+      },
+    },
+  },
+});
+
+const Banner = defineMultiStyleConfig({
+  baseStyle: definePartsStyle(({ backgroundColor, highlightColor }) => ({
+    base: {
+      alignItems: "flex-start",
+      bgColor: backgroundColor ? backgroundColor : null,
+      borderLeft: "4px",
+      borderColor: highlightColor ? highlightColor : null,
+      display: "flex",
+      flexDirection: "row",
+      gap: "xs",
+      justifyContent: "flex-start",
+      py: "s",
+      paddingEnd: "l",
+      paddingStart: {
+        base: "s",
+        md: "l",
+      },
+      position: "relative",
+      width: "100%",
+      a: {
+        color: "ui.typography.heading",
+        textDecoration: "underline",
```

Comment:
Did you add this to enforce the underline in the `Banner` component?  The `Link component is underlined by default, so I think we can rely on that and remove this attribute here.  Unless there was somethign else.  As it is, this is causing the underline to show as a fat solid line.  If we opt to include this here, we will also need to add the additional attributes that control the dotted style and the line weight.

Reply:
I did this because the link in the figma file has a solid underline and I thought that was intentional. I can remove this and use the default dotted underline.

Reply:
Figma can not render all of the CSS styles available for link underlines, so we included a disclaimer in the VDL referencing this discrepancy.

https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?type=design&node-id=79380-138172&mode=design&t=GjUSgxQmuNw7yPtS-11